src/include/partitioning/custom_part_check_generated.rb

```diff
@@ -549,6 +549,12 @@ def check_created_partition_table(targetMap, installation)
       ok
     end
 
+    def format_subvols(subvols)
+      # Extra space needed for the regexp
+      all = subvols.join(', ') + " "
```

Comment:
Not sure why this lone space is added at the end when later there is a strip() call